Training/Easy/chuck_norris.rb

```diff
@@ -27,25 +27,24 @@ def split_series(array)
     result
   end
 
-  def norris_parse(array)
-    result = ''
-    array.each do |a|
-      if a[0].zero?
-        result += '00 '
-      else
-        result += '0 '
-      end
-      result += '0' * a[1] + ' '
+  def norris_parse(series_array)
+    parsed = series_array.each_with_object('') do |a, result|
+      result << if a[0].zero?
```

Comment:
Instead of adding items with whitespace to a string object and then stripping it, you can add the same items without whitespace to an array object, after which you can use `join(' ')`